src/App.js

```diff
@@ -1,19 +1,47 @@
 import React from 'react';
+import { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [entryData, setEntryData] = useState(chatMessages);
+
+  const updateLikeData = updatedLike => {
+    const newLike = entryData.map(entry => {
+      if (entry.id === updatedLike.id) {
+        return updatedLike;
+      } else {
+        return entry;
+      }
+    });
+    setEntryData(newLike);
+  };
+
+  const likeCount = (entryData) => {
```

Comment:
This function already has access to `entryData` so it isn't necessary to pull it in through the function parameters.